tests/test_data/python-StructArray_dnfnc.spec

```diff
@@ -47,9 +47,9 @@ rm -rf %{pypi_name}.egg-info
 
 %files -n python2-%{pypi_name}
 %doc 
-%{python2_sitearch}/structarray.so
+%{python2_sitearch}/%{pypi_name}
```

Comment:
I'm unsure how is this relevant for the change. Maybe I'm missing something?

Reply:
[An integration test for StructArray](https://github.com/fedora-python/pyp2rpm/pull/149/commits/23bd40ac6bef5856516afe68e245f13c4a5d19c4#diff-7eb47fc18d4a6196f0940784484a5bf3R32) is now running with --no-venv switch. This was done in order to keep results of integration tests consistent in case virtualenv-api is not installed.